[Term Entry] Java Calendar Method getCalendarType() (#3760)
Codecademy/docs: content/java/concepts/calendar/terms/getCalendarType/getCalendarType.md

---

> Title: 'getCalendarType()'

Reviewer: ```suggestion
Title: '.getCalendarType()'
```

> Description: 'Retrieves the type or format of the calendar represented by the Calendar object, returns a string representing the calendar type'

Reviewer: ```suggestion
Description: 'Retrieves the type or format of the calendar represented by the Calendar object, returns a string representing the calendar type.'
```

Subjects:
  - 'Code Foundations'
  - 'Computer Science'
Tags:
  - 'Methods'
  - 'Classes'
  - 'Attributes'
CatalogContent:
  - 'learn-java'
  - 'paths/computer-science'
---

> The **getCalendarType()** method is available in the Calendar class in Java. It is used to retrieve the type or format of the calendar represented by the Calendar object. This method returns a string representing the calendar type, such as "gregory", "japanese", "buddhist", etc. This method does not accept any argument as a parameter.

Reviewer: ```suggestion
The **`.getCalendarType()`** method is available in the `Calendar` class in Java. It is used to retrieve the type or format of the calendar represented by the `Calendar` object. This method returns a string representing the calendar type, such as "gregory", "japanese", "buddhist", etc. This method does not accept any arguments.
```

## Syntax

> ```java

Reviewer: ```suggestion
```pseudo
```

> public String getCalendarType()

Reviewer: ```suggestion
someCalendar.getCalendarType()
```

```

> 

Reviewer: ```suggestion

- `someCalendar`: The given calendar to be addressed.
```

## Example

> 

Reviewer: There should be a brief introductory sentence before the code block

Author: Added short description. @caupolicandiaz I hope I've made all the changes.

```java
import java.util.Calendar;

public class MyClass {

>   public static void main(String[] args) {

Reviewer: make sure indentations are 2 spaces, not tabs

Author: Did it, I hope I did them all.

    // Create a Calendar instance representing the default calendar for the current locale
        Calendar calendar = Calendar.getInstance();

        // Get the calendar type and store it in a variable
        String calendarType = calendar.getCalendarType();

        // Display the calendar type
        System.out.println("Type of Calendar is: " + calendarType);
  }
}
```

> This example returns the calendar type or class name of that calendar type:

Reviewer: ```suggestion
This example returns the following output:
```

---
Title: 'getCalendarType()' 
Description: 'Retrieves the type or format of the calendar represented by the Calendar object, returns a string representing the calendar type' 
Subjects: 
  - 'Code Foundations'
  - 'Computer Science'
Tags: 
  - 'Methods'
  - 'Classes'
  - 'Attributes'
CatalogContent: 
  - 'learn-java'
  - 'paths/computer-science'
---

> The **get CalendarType** method is a function available in the Calendar class in Java. It is used to retrieve the type or format of the calendar represented by the Calendar object. This method returns a string representing the calendar type, such as "gregory", "japanese", "buddhist", etc. This method does not accept any argument as a parameter.

Reviewer 2: ```suggestion
The **getCalendarType()** method is available in the Calendar class in Java. It is used to retrieve the type or format of the calendar represented by the Calendar object. This method returns a string representing the calendar type, such as "gregory", "japanese", "buddhist", etc. This method does not accept any argument as a parameter.
```

## Syntax

```java
public String getCalendarType()
```

## Example

> No parameter is passed to this method.

Reviewer 2: There already is a mention of `.getCalenderType()` not requiring any parameter. So, please remove this sentence.

Author: @ishg-153 Removed the redundant sentence.

```java
import java.util.Calendar;

public class MyClass {
  public static void main(String[] args) {
    // Create a Calendar instance representing the default calendar for the current locale
        Calendar calendar = Calendar.getInstance();

        // Get the calendar type and store it in a variable
        String calendarType = calendar.getCalendarType();

        // Display the calendar type
        System.out.println("Type of Calendar is: " + calendarType);
  }
}
```
This example returns the calendar type or class name of that calendar type:

> ```java

Reviewer 2: ```suggestion
```shell
```

Since this is the output for the above example, it should be enclosed as suggested.
